de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5476950_sys_gh2956_drop_allowconsolidateinvoice.sql

```diff
@@ -0,0 +1,7 @@
+--set default aggregation for partner with activated consolidation
+update c_bpartner
```

Comment:
hi @metasnw i think I aggree with what you wrote..
so imho we should drop the column (and create a backup, as teo suggested),
but not add an so_invoice_aggregation_id to partners that have none right now.

also with this search:
![image](https://user-images.githubusercontent.com/1264789/32767327-26c6483c-c913-11e7-806a-2b508c900b45.png)

I get a lot of commented-out code..so while were are here, i propose to remove that commented-out code too..
maybe we can ask @metas-rc to finish this task according to the proposal?

Reply:
update @metasnw :
i yesterday asked @metas-rc to add it to her todo-list